compiler/rustc_target/src/spec/targets/aarch64_unknown_trusty.rs

```diff
@@ -0,0 +1,34 @@
+// Trusty OS target for AArch64.
+
+use crate::spec::{LinkSelfContainedDefault, PanicStrategy, RelroLevel, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "aarch64-unknown-unknown-musl".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: Some("ARM64 Trusty".into()),
+            tier: Some(2),
```

Comment:
This seems like a mistake. It's supposed to be tier 3, isn't it?

Reply:
Indeed my bad, missed it during my review.

@randomPoison may I ask you to send a PR fixing the target tier here and in the other target.

Reply:
Ah yup this was a copy-paste error. I've put up https://github.com/rust-lang/rust/pull/129712 to fix it.